package.json

```
@@ -73,6 +111,12 @@
           ],
           "default": "ask",
           "description": "Controls how NPL instruction files for AI assistants are handled"
+        },
+        "noumena.cloud.portalUrl": {
+          "type": "string",
+          "default": "https://portal.pre.noumena.cloud",
```

Device flow has not been deployed to prod yet, so this will have to be changed once it's live

```suggestion
          "default": "https://portal.noumena.cloud",
```

---

Added a config for the keycloak url, so actually for this PR we no longer require the portal URL, but the point still stands and I'll leave it in here.